examples/open-meanstack/README.md

```diff
@@ -0,0 +1,39 @@
+Meanstack Example
+-----------------
+
+### Example Reference:
+https://kubernetes.io/docs/getting-started-guides/meanstack/
+
+```
+$ opencompose convert -f opencompose-meanstack.yml
+created file "mongo-service.yaml"
+created file "mongo-deployment.yaml"
+created file "web-service.yaml"
+created file "web-deployment.yaml"
+```
+
+### Deploy the example on kubernetes
+
+```
+$ kubectl create -f .
+deployment "mongo" created
+service "mongo" created
+deployment "web" created
+service "web" created
+```
+
+```
+$ kubectl get deployments
+NAME      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+mongo     1         1         1            1           5m
+web       2         2         2            2           5m
+```
+
+```
+$ kubectl get services
+NAME          CLUSTER-IP   EXTERNAL-IP   PORT(S)           AGE
+mongo         10.0.0.33    1.2.3.4       27017/TCP         6m
+web           10.0.0.154   1.2.3.5       3000/TCP          6m
```

Comment:
question out of curiousity, did you actually get the external ip address? I was trying it on the minikube vm and that field remained pending?